one_fm/grd/doctype/medical_insurance/test_medical_insurance.py

```diff
@@ -3,8 +3,9 @@
 # See license.txt
 from __future__ import unicode_literals
 
-# import frappe
+#import frappe
 import unittest
 
+
```

Comment:
We can undone these changes, no need to track this

Reply:
done..